src/test/regress/sql/multi_test_helpers_shadow.sql

```diff
@@ -0,0 +1,106 @@
+-- File to create functions and helpers needed for subsequent tests
```

Comment:
1. It is not clear that file this file exists, make it clear.
2. I see that this is multi_test_helpers.sql with couple of UDFs removed. First, why do we need to do that? Second, if we really need to do that, I suggest we create two files `multi_test_helpers_common.sql` and `multi_test_helpers_local_tests.sql`, and use the `\include` to avoid repeating definitions. See https://www.postgresql.org/docs/current/app-psql.html